cyper/src/body.rs

```diff
@@ -191,3 +191,17 @@ impl hyper::body::Body for ResponseBody {
         }
     }
 }
+
+#[cfg(feature = "stream")]
+impl Stream for ResponseBody {
+    type Item = Result<Bytes, crate::Error>;
+
+    fn poll_next(self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Option<Self::Item>> {
+        match hyper::body::Body::poll_frame(self, cx).map_ok(|frame| frame.into_data())? {
+            Poll::Ready(None) => Poll::Ready(None),
+            Poll::Ready(Some(Ok(bytes))) => Poll::Ready(Some(Ok(bytes))),
+            Poll::Ready(Some(Err(_))) => Poll::Pending,
```

Comment:
Hm, why?

Reply:
I forgot about it really, will fix :+1:

Reply:
In this case if `frame.into_data` returns `Err`, the error value is the frame itself. It'll return `Err` only if the current frame is a trailer. It could make sense to return an error here but idk really, I can clearly see how someone would halt execution whenever the first error comes in, even though it's the expected behavior.

 I'm not exactly familiar with the semantics of these, but could you confirm when do trailers come in ? I'm guessing they only come at the end but I'm not so sure really :) If they arrive last we can return `Poll::Ready(None)` here instead.

Also I chose to only return `Bytes` and not `Frames` which could mitigate this but I wanted to replicate the reqwest implementation which also returns a stream of `Bytes`

Reply:
I see, I'll do the same then :+1: